qpc/messages.py

```diff
@@ -116,6 +116,17 @@
 SCAN_RESTARTED = 'Scan "%s" restarted.'
 SCAN_CANCELED = 'Scan "%s" canceled.'
 
+REPORT_SUMMARY_ID_HELP = 'Fact collection identifier.'
```

Comment:
Supposed to take the Scan ID and pull the fact collection ID from there

Reply:
Ok.  Fixed.